docker-compose.yml

```diff
@@ -28,6 +28,7 @@ services:
     command: >
       --datadir dev-chain/
       --http
+      --http.api eth,web3,personal,net
```

Comment:
Not sure why these aren't enabled by default in dev mode. Possibly people are running dev nodes in production and geth is protecting them? 🤯

I got these flags from https://geth.ethereum.org/docs/getting-started/dev-mode. They might not actually be needed for my use-case, as it turns out. But I think they're worth including anyway.